kythe/go/platform/kzip/kzip.go

```diff
@@ -470,8 +492,8 @@ func (w *Writer) AddUnit(cu *apb.CompilationUnit, index *apb.IndexedCompilation_
 		return digest, ErrUnitExists
 	}
 
-	if w.encoding == JSON || w.encoding == BOTH {
-		f, err := w.zip.CreateHeader(newFileHeader("root", "units", digest))
+	if w.encoding == EncodingJSON || w.encoding == EncodingBoth {
```

Comment:
Any compelling reason not to make encoding a bit field instead of literal integers?
```
EncodingJSON = 1,
EncodingProto = 2
EncodingAll = EncodingJSON | EncodingProto
```
Just to have the zero-value be "all"?